cni/plugin.go

```diff
@@ -198,3 +201,27 @@ func (plugin *Plugin) UninitializeKeyValueStore() error {
 
 	return nil
 }
+
+// Lock key-value store for Windows Runtime. This function is being used for locking access to TelemetryService start.
+func (plugin *Plugin) LockKeyValueStore() error {
+	if runtime.GOOS == "windows" {
```

Comment:
can we move this condition to a place where we call LockKeyValueStore()

Reply:
addressed in the new commit